meadow-client/src/Semantics.purs

```diff
@@ -486,15 +288,11 @@ observationParser = do
       choice <- bigIntParser
       person <- bigIntParser
       choice2 <- bigIntParser
-      pure (ChoseThis { choice
-                      , person
-                      } choice2)
+      pure (ChoseThis (IdChoice { choice, person }) choice2)
     "ChoseSomething" -> do
       choice <- bigIntParser
       person <- bigIntParser
-      pure (ChoseSomething { choice
-                           , person
-                           })
+      pure (ChoseSomething (IdChoice { choice, person }))
```

Comment:
There's an `idParser` already available, but it will parse commas. Should we be using it instead?

Reply:
see above, I think this only comes up in the PR because of purty

Reply:
👍